packages/collector/src/logger.js

```diff
@@ -35,37 +35,66 @@ exports.init = function init(config, isReInit) {
     // A bunyan or pino logger has been provided via config. In either case we create a child logger directly under the
     // given logger which serves as the parent for all loggers we create later on.
     parentLogger = config.logger.child({
-      module: 'instana-nodejs-logger-parent',
-      __in: 1
+      module: 'instana-nodejs-logger-parent'
     });
   } else if (config.logger && hasLoggingFunctions(config.logger)) {
-    // A custom non-bunyan logger has been provided via config. We use it as is.
+    // A custom non-bunyan/non-pino logger has been provided via config. We use it as is.
     parentLogger = config.logger;
   } else {
-    // No custom logger has been provided via config, we create a new bunyan logger as the parent logger for all loggers
+    // No custom logger has been provided via config, we create a new pino logger as the parent logger for all loggers
     // we create later on.
-    parentLogger = bunyan.createLogger({
+    // @ts-ignore
+    parentLogger = pino({
       name: '@instana/collector',
       thread: threadId,
```

Comment:
issue: There is no field thread

https://github.com/pinojs/pino/blob/main/pino.d.ts

Reply:
issue: @ts-ignore is not right

Reply:
https://github.com/instana/nodejs/pull/1447/commits/ffdc7750aa76a391c3e872b2b9936b01c7975ad3